### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by jasmine-bardi** [community/community#33812] [Other Feature Feedback, Questions, & Ideas] [answered]
Please I wanted to login into my github account and I saw a message that my account was suspended  pls how can I get my account back because all my documents are there I had to make a new account to lodge a complain
This is the email for the account  [redacted due to PII]
![72E38827-A654-4D9C-87E5-954FE3404433](https://user-images.githubusercontent.com/114147306/191715559-ebfb4262-d256-499e-915d-1eeee63cf070.png)

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Good luck!

**Reply by ghost** [reply]
Same Here. They Shouldnt have done this without any warnings or Notice. And Now They are not even responding on the support.

**Reply by queenofcorgis** [reply]
@shekhawatanand - appreciate your patience. Our Support team will get back to you in the order they received your ticket depending on how many cases are ahead of yours. Good luck!

**Reply by dungng1321** [reply]
Mee too .Did you get your account back?

**Reply by engsiam** [reply]
help me my old account @siam786 please back

**Comment by aynaash** [comment]
Hi my account got suspended and I dont know why I contacted support but got no response I had to create a another account to use for the time being and I want my account back so what the best way forward, thanks!
